sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/multi/MultiServiceManager.java

```diff
@@ -187,8 +189,8 @@ public void uninstallService(String serviceName) {
     public void uninstallServices(Collection<String> finishedServiceNames) {
```

Comment:
Could be protected, no?

Reply:
No particular reason to, theres a public `uninstallService` which is just wrapping this call. Its conceivable that there would be some utility in marking multiple services for uninstall atomically.